draft-wussler-openpgp-pqc.md

```diff
@@ -1057,47 +979,48 @@ scheme is as follows:
 
 ## Packet specifications
 
-### Public-Key Encrypted Session Key Packets (Tag 1) {#ecc-kyber-pkesk}
+### Public-Key Encrypted Session Key Packets (Tag 1) {#ecc-mlkem-pkesk}
 
 The algorithm-specific fields consists of:
 
  - A fixed-length octet string representing an ECC ephemeral public key in the
    format associated with the curve as specified in {{ecc-kem}}.
 
- - A fixed-length octet string of the Kyber ciphertext, whose length depends on
-   the algorithm ID as specified in {{tab-kyber-artifacts}}.
+ - A fixed-length octet string of the ML-KEM ciphertext, whose length depends
+   on the algorithm ID as specified in {{tab-mlkem-artifacts}}.
 
- - The one-octet algorithm identifier, if it was passed (in the case of a v3 PKESK packet).
+ - The one-octet algorithm identifier, if it is passed (in the case of a v3
+   PKESK packet).
 
  - A variable-length field containing the wrapped session key:
 
    - A one-octet size of the following field;
 
-   - The wrapped session key represented as an octet string, i.e.,
-     the output of the encryption procedure described in
-     {{ecc-kyber-encryption}}.
+   - The wrapped session key represented as an octet string, i.e., the output
+     of the encryption procedure described in {{ecc-mlkem-encryption}}.
 
-Note that unlike most public-key algorithms, in the case of a v3 PKESK packet, the symmetric algorithm identifier is not encrypted.
-Instead, it is prepended to the encrypted session key in plaintext.
-In this case, the symmetric algorithm used MUST be AES-128, AES-192 or AES-256 (algorithm ID 7, 8 or 9).
+Note that unlike most public-key algorithms, in the case of a v3 PKESK packet,
+the symmetric algorithm identifier is not encrypted.  Instead, it is prepended
+to the encrypted session key in plaintext.  In this case, the symmetric
+algorithm used MUST be AES-128, AES-192 or AES-256 (algorithm ID 7, 8 or 9).
 
-### Key Material Packets {#kyber-ecc-key}
+### Key Material Packets {#mlkem-ecc-key}
 
 The algorithm-specific public key is this series of values:
 
  - A fixed-length octet string representing an EC point public key, in the
    point format associated with the curve specified in {{ecc-kem}}.
 
- - A fixed-length octet string containing the Kyber public key, whose length
-   depends on the algorithm ID as specified in {{tab-kyber-artifacts}}.
+ - A fixed-length octet string containing the ML-KEM encapsulation key, whose
```

Comment:
I am not sure about the change to "encapsulation/decapsulation key" from "public/private key". Otherwise, in order to be consistent, we would have to eliminate all uses of "public/private key" and replace it with "encryption/decryption key" or "verification/signature key". Does anyone see a specific reason why KEM keys should be handled differently in this respect?